modules/compliance-apply-remediation-for-customized-mcp.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: PROCEDURE
+[id="complianc-operator-apply-remediation-for-customized-mcp"]
+= Applying remediation when using customized Machine Config Pools
```

Comment:
I think machine config pools should be lower case. Is that term always in caps?

Reply:
Further, the ISG suggests avoiding using Config, but I think this would fit the exceptions. You may need to ask in the team docs channel though since both style guides do not mention this yet.

Reply:
Addressed

Reply:
will confirm regarding usage of config on team-docs channel

Reply:
Hi @EricPonvelle I confirmed on the team-docs channel about the usage of `config`. Here it would be the valid case for using the word `config`